lib/vector-core/src/event/vrl_target.rs

```diff
@@ -26,7 +28,11 @@ pub enum VrlTarget {
     // `LogEvent` is essentially just a destructured `event::LogEvent`, but without the semantics
     // that `fields` must always be a `Map` variant.
     LogEvent(Value, EventMetadata),
-    Metric { metric: Metric, value: Value },
+    Metric {
+        metric: Metric,
+        value: Value,
+        multi_value_tags: bool,
```

Comment:
I'm somewhat disappointed this flag has to be carried around at an assignment value level. In the grand scheme of things, adding another word to this enum isn't going to be a big deal (the `Metric` type is large enough), but I had naïvely envisioned it could be managed at a higher layer somehow.